validation/views.py

```diff
@@ -153,4 +163,55 @@ def add_cors_headers(response):
     return response
 
 
+def login(request):
+    """
+    Serves the login page and sets cookies on successful login
+    """
+
+    if request.method == "GET":
+        form = Login(request.GET)
+        if form.is_valid():
+            username = form.cleaned_data["username"]
+            password = form.cleaned_data["password"]
+            user = authenticate(request, username=username, password=password)
+            if user is not None:
+                django_login(request, user)
+                response = HttpResponseRedirect("/")
+                return response
+    form = Login()
+    context = dict(form=form)
+    return render(request, "validation/login.html", context)
+
+
+def register(request):
+    """
+    Serves the login page and sets cookies on successful login
```

Comment:
Copy-paste of docstring from previous method?

Reply:
Good catch!